libpod/container_internal_linux.go

```diff
@@ -385,6 +385,16 @@ func (c *Container) generateSpec(ctx context.Context) (*spec.Spec, error) {
 			g.AddLinuxGIDMapping(uint32(0), uint32(0), uint32(1))
 		}
 	}
+
+	for _, i := range c.config.Spec.Linux.Namespaces {
+		if i.Type == spec.UTSNamespace {
```

Comment:
Shouldn't we check for path here? To see if a path to the UTS namespace was given?

Reply:
I've only moved the existing chunk.

When we join an existing container these values are overwritten in `addNamespaceContainer`